test/test_encryption.py

```diff
@@ -2335,6 +2335,31 @@ def test_02_success(self):
         self.client_encryption.create_data_key("aws", self.master_key)
 
 
+# https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/tests/README.rst#8-bypass-spawning-mongocryptd
+class TestBypassMongocryptWithSharedLibrary(EncryptionIntegrationTest):
```

Comment:
Let's add this to the existing TestBypassSpawningMongocryptdProse class instead.